OperatingSystem-Services/Service-SOC/WazuhConfigs/Init_Agent_Config.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# TODO: Currentlyl just appends everything to the config file, would be nice to not add all the rules if they are already there
+# OSSEC CONFIF FOR AUDITD IS CURRENTLY BROKEN!!!
+# Created by VA
+## Must run as superuser ##
+
+if [ "$EUID" -ne 0 ]; then 
+  echo "ERROR: Must run as superuser"
+  exit
+fi
+
+sed -i.bak 's|<interval>1h</interval>|<interval>7min</interval>|g' /var/ossec/etc/ossec.conf
+sed -i.bak 's|<ports all="no">yes</ports>|<ports all="yes">yes</ports>|g' /var/ossec/etc/ossec.conf
+sed -i.bak '/<!-- Log analysis -->/a \\n  <localfile>\n    <log_format>audit</log_format>\n    <location>/var/log/audit/audit.log</location>\n  </localfile>' /var/ossec/etc/ossec.conf
+
+
```

Comment:
You could add a nice comment if you wanted to. Explaining why these ones in particular